### ISSUE TITLE
A way to make strings pop

### ISSUE THREAD

**Issue by omar2205** [argyleink/Kung-Fury-Theme#5]
![image](https://user-images.githubusercontent.com/1373867/145261290-0c29af03-b5f4-4011-b737-e526f45ac6d2.png)

A way to change "word" color?

**Comment by argyleink** [comment]
'world' color? 

i have all strings as the most contrasting color at the moment

**Comment by omar2205** [comment]
@argyleink, yeah, I want to highlight the whole string like normal, I also, want to think about a good color to match the theme (for the string)

**Comment by omar2205** [comment]
![image](https://user-images.githubusercontent.com/1373867/145276088-3ed2239b-d221-41af-a15e-4991a07c89c1.png)

As you can see, the string "lib" is colored, but "hello" isn't. Also, the quotes are too dark.

**Comment by argyleink** [comment]
oh yeah, the import string appears to be highlighted as a constant, i can check that out 👍🏻  

the quotes are only dark if they're correct.. and it's part of the "noise reduction" value of this theme. i dont see those changing

**Comment by omar2205** [comment]
Is there a way to give strings different colors?

**Comment by argyleink** [comment]
yep, but there's a bunch of types and contexts https://github.com/argyleink/Kung-Fury-Theme/blob/master/themes/Kung%20Fury.tmTheme#L16
